tests/phpunit/tests/blocks/editor.php

```diff
@@ -27,12 +27,18 @@ public function set_up() {
 		global $wp_rest_server;
 		$wp_rest_server = new Spy_REST_Server();
 		do_action( 'rest_api_init', $wp_rest_server );
+
+		global $post_ID;
+		$this->orig_post_id = $post_ID;
```

Comment:
Looks like this is throwing errors in the PHP 8.2 actions:

<img width="641" alt="image" src="https://github.com/WordPress/wordpress-develop/assets/14988353/c66c647a-0d35-414f-a8fb-53de0ae582e9">

Would the fix be to define `orig_post_id` at the top of this class?

Reply:
Looks good to me!